### ISSUE TITLE
New material for Chapter 5.6 pending from chapter 4?

### ISSUE THREAD

**Issue by willclarkson** [LSSTScienceCollaborations/ObservingStrategy#321]
Hi @AshishMahabal and @lmwalkowicz -

@pmmcgehee is contributing some interesting material on detecting Young Stars through Variability. Because of the way Peregrine has arranged his contribution, in the current whitepaper version this now appears as Section 4.4 (since Pull Request #391). @pmmcgehee has promised more input later today (Monday), so we are holding off on making changes (like refactoring) for a few hours. For the moment, however, you can view his material in Section 4.4 (starting on page 66 in the current version).  

@akvivas and myself think this could drop very nicely into your Section 5.6 (detecting young stars through variability) that @phartigan and Christopher Johns-Krull are listed as authoring. 

When Peregrine has finished his contribution, do you have any objection to myself or @akvivas moving that material (suitably edited by us first) across to Section 5.6 and adding @pmmcgehee as an author to that section?

Thanks!!

Will

**Comment by pmmcgehee** [comment]
I am glad to hear that there are additional authors supplying material pertaining to young stars. My first draft was essentially the Science Book section dealing with FU Orionis stars. If @phartigan has included more on WTTS and CTTS detection that would be great.

I have not created metrics, but these would follow previous project work on periodic and non-periodic variable stars.

**Comment by AshishMahabal** [comment]
@willclarkson that will be wonderful (merging @pmmcgehee 's part in to 5.6). Thanks!

**Comment by willclarkson** [comment]
Thanks @AshishMahabal ! 

I believe @phartigan has been working on new material for your Section 5.6 separately, over the past few days. I wasn't aware that that material was so close to submission when I opened this issue this morning. 

To minimize the chance of anything being accidentally overwritten, I recommend waiting until @phartigan has had a chance to submit the pull request for the content on which he has been working. Then we can merge the two versions of the Section. 

@AshishMahabal and @lmwalkowicz , the SFH material from the Galaxy chapter is the file ./Milkyway/MW_SFH.tex in the repository. 

cc @akvivas

**Comment by pmmcgehee** [comment]
Sounds like a good plan...

Peregrine

On Mon, Apr 25, 2016 at 6:40 PM, Will Clarkson <notifications@github.com>
wrote:

> Thanks @AshishMahabal <https://github.com/AshishMahabal> !
>
> I believe @phartigan <https://github.com/phartigan> has been working on
> new material for your Section 5.6 separately, over the past few days. I
> wasn't aware that that material was so close to submission when I opened
> this issue this morning.
>
> To minimize the chance of anything being accidentally overwritten, I
> recommend waiting until @phartigan <https://github.com/phartigan> has had
> a chance to submit the pull request for the content on which he has been
> working. Then we can merge the two versions of the Section.
>
> @AshishMahabal <https://github.com/AshishMahabal> and @lmwalkowicz
> <https://github.com/lmwalkowicz> , the SFH material from the Galaxy
> chapter is the file ./Milkyway/MW_SFH.tex in the repository.
>
> cc @akvivas <https://github.com/akvivas>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-214582009>
>



-- 
Dr. Peregrine M. McGehee
Santa Clarita, CA    (626) 993-4199

**Comment by drphilmarshall** [comment]
Pat, are you aware of the timeline this week? Science sections are getting audited tomorrow, so that a "Final Draft" can be prepared for the editors to look at on Thursday morning. If you are not ready to submit a pull request yet, that's OK - I will simply audit what I *can* see, in the Galaxy chapter, and trust that yours and Peregrine's stuff can be merged together later, somehow. Thanks all!

**Comment by pmmcgehee** [comment]
Good luck, all. I've finished my contributions, at LAX waiting on a Paris
flight....

Best wishes
Peregrine
On Apr 26, 2016 3:49 PM, "Phil Marshall" <notifications@github.com> wrote:

Pat, are you aware of the timeline this week? Science sections are getting
audited tomorrow, so that a "Final Draft" can be prepared for the editors
to look at on Thursday morning. If you are not ready to submit a pull
request yet, that's OK - I will simply audit what I *can* see, in the
Galaxy chapter, and trust that yours and Peregrine's stuff can be merged
together later, somehow. Thanks all!

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-214911380>

**Comment by phartigan** [comment]
Hi,

Thought I submitted this last night and was waiting to see what  
response there was.
I can finish up the paragraphs if that's what you're looking for. I will
need to know how to embed figures and add references in your preferred format.
Let me know.

ph

**Comment by willclarkson** [comment]
Hi @phartigan - aha, it looks like your file went to the ObservingStrategy/files directory instead of the whitepaper itself, so we probably missed it (I know I did!). Thanks for this correction! 

I recommend putting the content in your lsst_blurb.txt into youngstars.tex and then submitting to the whitepaper repository as a pull request. I'm happy to do this action if you like.  

Figures are stored in the subdirectory figs/ of the whitepaper/ repository. In the Milky Way chapter we have been using includegraphics to put in figures. For Chapter 5 I think the file whitepaper/variables.tex is a good example. Ditto for references; we have been using bibtex.

cc @AshishMahabal @lmwalkowicz @akvivas 

Thanks!! -- Will

**Comment by phartigan** [comment]
Hi Will,

I'm not surprised. I've not used github before and I've found its setup
rather arcane, though I'm sure it is second nature once you get used
to it. Go ahead and pull/merge whatever to update youngstars.tex.

Since there doesn't seem to be time for feedback, let me just
forge ahead and make this a self-contained document, with figures
and references. Then I'll either try the pull thing again, or
if you are ok with it, I could send you the final file with figures
and you can insert it. The figures are done, and it is mainly
doing references, captions and what-not. Do you want the figures
as eps for includegraphics?

Pat

**Comment by willclarkson** [comment]
Thanks @phartigan - it sounds like making the self-contained .tex file is the way to go. I will be able to merge that as youngstars.tex tomorrow morning (Weds Apr 27th - I'm in the EDT time zone) if you like (unless @AshishMahabal or @lmwalkowicz want to). 

Figures: in Chapter 4 we've been using .png and .pdf for figure files. I don't think there are any examples of .eps figures in the repository, so I don't know if .eps are deemed "acceptable" by the overall editors. It's probably safer to convert to .png or pdf if you can.

cc @akvivas 

Thanks!!

Will

**Comment by phartigan** [comment]
pdf it is!

I'll work on this and email it by tomorrow morning.

ph

**Comment by AshishMahabal** [comment]
@willclarkson, please go ahead (@phartigan , @akvivas , @lmwalkowicz )

**Comment by phartigan** [comment]
Dear Will,

Attached are
1. the new youngstars.tex section
2. Three pdf figures.

This should all be self-contained. I also made a pull request
but I don't know if that worked. In order to attach the three figures
I had to close the pull request so perhaps that killed it.
Please insert this if needed.

Thanks

ph

**Comment by willclarkson** [comment]
Hi Pat - I have added your new *youngstars.tex* into the repository. Thanks
for the contribution!!! A few notes:

*1. *I recommend bringing down the repository version of youngstars.tex
before making further edits, I had to make a few minor alterations (close
braces, fix runaway \bf etc) to fix compilation errors. If you have not yet
"forked" the repository, you can get the latest version of the entire
repository as follows:

1.1. Navigate to
https://github.com/LSSTScienceCollaborations/ObservingStrategy

1.2. Click the "fork" button on the top-right of the page;

1.3. In a terminal, "git clone" the SSH URL that appears in the page of
your fork. This will bring down the latest version of the repository in
your own "fork" to make changes.

*2. *In the repository, the figures are in the new directory
./figs/starFormation/

*3. *The bibitems you provided at the end, should go into the file
references.bib . For the moment I commented them out to aid compilation.

*4. *You might want to take a look at the three figures as they render in
the document, there may be syntax that could be added to the
includegraphics calls that place the figures on the page a little more
efficiently.

Great stuff!!

Will



--

Dr. Will Clarkson
Assistant Professor of Physics and Astronomy
University of Michigan-Dearborn

**Comment by drphilmarshall** [comment]
Thanks all - I see a very nice young stars section 5.7, ready for editing together with 4.4. Thanks for the feedback, @willclarkson - I'll consider the figure formatting fix part of this issue. @willclarkson are you working on bringing 4.4 into 5.6? I will skip over 4.4 in my audit, on the assumption that @pmmcgehee 's content will end up in 5.6 thanks to your efforts.

@phartigan Well done for bending git and GitHub to your will! :-) And thanks every so much for hustling your section out - much appreciated.

**Comment by phartigan** [comment]
> @phartigan Well done for bending git and GitHub to your will! :-)  
> And thanks every so much for hustling your section out - much  
> appreciated.

I'm afraid GitHub won our cage match...

**Comment by phartigan** [comment]
Hi Will,

I did forks and branches and commits at some point, but I'm never sure
where I am in there, and think it got lost. Not surprised there were some
latex glitches, as I didn't actually see it rendered, lacking the  
front material that
defines the latex document. So I just typed everything in and hoped it
was close enough to being readable. The figures ought to span the width of
the page. A standalone template with the front matter that defines font,
page size, line spacing and what-not would have been helpful.

I'm getting the sense that we should all sit tight until the merging is
complete, correct?

ph

> Hi Pat - I have added your new *youngstars.tex* into the repository. Thanks
> for the contribution!!! A few notes:
>
> *1. *I recommend bringing down the repository version of youngstars.tex
> before making further edits, I had to make a few minor alterations (close
> braces, fix runaway \bf etc) to fix compilation errors. If you have not yet
> "forked" the repository, you can get the latest version of the entire
> repository as follows:
>
> 1.1. Navigate to
> https://github.com/LSSTScienceCollaborations/ObservingStrategy
>
> 1.2. Click the "fork" button on the top-right of the page;
>
> 1.3. In a terminal, "git clone" the SSH URL that appears in the page of
> your fork. This will bring down the latest version of the repository in
> your own "fork" to make changes.
>
> *2. *In the repository, the figures are in the new directory
> ./figs/starFormation/
>
> *3. *The bibitems you provided at the end, should go into the file
> references.bib . For the moment I commented them out to aid compilation.
>
> *4. *You might want to take a look at the three figures as they render in
> the document, there may be syntax that could be added to the
> includegraphics calls that place the figures on the page a little more
> efficiently.
>
> Great stuff!!
>
> Will
>
>
>
> --
>
> Dr. Will Clarkson
> Assistant Professor of Physics and Astronomy
> University of Michigan-Dearborn
>
>
>

**Comment by drphilmarshall** [comment]
While you're waiting, Pat, have you seen my blockbuster movie "Getting
Started with git and GitHub"? It's linked from the following test repo,
which you are more than welcome to fork and play around with:

https://github.com/drphilmarshall/GettingStarted

Happy to hang out by skype sometime if you ever want someone to look over
your shoulder as you continue the fight!

**Comment by phartigan** [comment]
I went through their tutorial, created a Hello World, forked it, made
branches, and then had a bunch of copies lying around in various directories
with changes here and there. It seemed like a complicated way to do something
rather simple. In the case of LSST it wasn't clear to me who owned what, who
approved what, how long the delay would be before I could view changes etc.
Since there was a deadline I decided to focus on content and let others
manage the tool.

I'll look at your blockbuster.

ph

**Comment by willclarkson** [comment]
Hi @phartigan - just to reply to your question from a few comments ago: I do not expect to touch youngstars.tex again today, I will be working on Chapter 4 issues. 

If you are comfortable enough with github to e.g. change figure boundaries or add the authors to references.bib , please go ahead and do so.

**Comment by phartigan** [comment]
ok. Am about done with the video. I'll probably give the figures a shot.
I guess I make a fork, and then I edit, commit, and should see  
changes, and then
submit a pull request.  My understanding is that clones are only needed if
I'm working locally off-line. Doesn't look like I need to create branches.
Is that correct?

ph

**Comment by StephenRidgway** [comment]
Hi,

I’m probably the least adept with GIT of any of you.  I don’t know if our setups are similar, but when I get confused I find it clears things up to delete my current files and start over, as follows:


Start in github directory.

	git clone git@github.com:LSSTScienceCollaborations/ObservingStrategy.git

 edit files in the “whitepaper” directory


 	To typeset locally, use "make" in the whitepaper directory.


 git commit -am "comment"


 git status


 git push origin master


If I’m doing anything dangerous here, hopefully somebody will point it out, but it seems to do what I need.

Steve

**Comment by drphilmarshall** [comment]
No, you've got it Steve. The only difference is that Pat, you should clone
using the .git address of your fork,

    git clone git@github.com:phartigan/ObservingStrategy.git

and then push your edits there, so you can submit a pull request on the
GitHub site. To keep up with the base repo, you'll want to connect to that
as well, via

    git remote add base git@github.com:LSSTScienceColl
aborations/ObservingStrategy.git

Then you can

    git pull base master

to bring your local repository up to date with the primary version, and

    git push origin master

to publish your edits on your fork.

**Comment by phartigan** [comment]
Hmm.

As these are all latex documents, to see how things look presumably I'll run
latex on some master file that sources the various .tex and uses some style
file that it finds in the repository. I guess that is what `make' does?
And the output is a .dvi or .eps or .pdf? Does my local latex version  
matter, or
is that bundled in the repo as well, with the fonts and everything?

So the purpose of the fork is that I own it and can put edits there.  
Ok, but then
why not just edit on the fork directly, why clone? Or can't I run  
'make' directly
on the fork, just on a local clone? Or is it the convenience of using a real
editor and unix commands and not some git-gui with green buttons on the web?

ph

**Comment by drphilmarshall** [comment]
@phartigan 

> As these are all latex documents, to see how things look presumably I'll run
latex on some master file that sources the various .tex and uses some style
file that it finds in the repository. I guess that is what `make' does?

Yes.

> And the output is a .dvi or .eps or .pdf? Does my local latex version  
matter, or is that bundled in the repo as well, with the fonts and everything?

Output is PDF. Your local latex version is probably OK, give it a try.

> So the purpose of the fork is that I own it and can put edits there.  OK, but then why not just edit on the fork directly, why clone? Or can't I run  'make' directly on the fork, just on a local clone? Or is it the convenience of using a real editor and unix commands and not some git-gui with green buttons on the web?

Exactly: when working with latex you need to be able to compile the document locally to make sure it has no errors in it. git enables you to save old versions, and to keep up with the many contributions being made by your co-authors. GitHub provides a central server, offsite, and a communication system to support collaboration (which we are using now).

**Comment by phartigan** [comment]
ok. I was just working from the fork and found it very annoying - cumbersome
cuts and pastes in their web editor, couldn't see the pdf, confusing branches,
terminology. This way makes a bit more sense, though I suspect I'll have
to go through the ssh-keygen steps.

ph

**Comment by drphilmarshall** [comment]
It's worth it, to save typing your password all the time. You're right, GitHub makes a lot more sense if you already know about and/or use git from the command line, in place of svn or cvs. If you encounter GitHub before git, then I can see how you might think that the web interface was the main event, rather than just being a nice way to show what's going on with a project that is using git. In some ways it would be helpful of GitHub disabled their in-browser editor!

**Comment by phartigan** [comment]
Right. ok, in the next few days I'll find time to experiment
with it that mode and shake off my vaguely negative impression of the  
web setup.

It seems like this project is sort of on the boundary of needing all this.
In the limit that we all exclusively edit our own sections, then none
of this is needed - we could just email our pieces to the PI and look
at the latest pdf when posted. But I can see where you have issues
if you have multiple editors for one piece of prose, or a really long
computer code with lots of subroutines which is probably where this
shines.

ph

**Comment by drphilmarshall** [comment]
Hi all: 

At the editorial meeting ( #293 ) we decided that @lmwalkowicz would take over Section 5.4 "Discovery and Characterization of Young Stellar Populations", and manually merge together @phartigan and @pmmcgehee 's work - and that this would also count as this section's internal review. Such efficiency! Thanks for doing this, Lucianne. Feel free to ask both @phartigan and @pmmcgehee for clarifications any time (and me too I guess, for doing the initial cut and paste job...)

**Comment by phartigan** [comment]
Not sure if you want to add this, but we have CTIO 4-m DecCam images,
4x60sec at z of Carina, and find significant photometric confusion
for less than 5% of the visible stars down to a limit where the photometry
gets noisy. These images will be slightly deeper than those obtained
with standard LSST cadences. It is hard to quantify this sort of thing
without test images because of extinction. May be worth mentioning, or not.

ph

**Comment by phartigan** [comment]
Hi,

A couple of questions...

I seem to have the ssh-keygen working. and successfully downloaded my  
github fork
to my local machine with the clone command. However...

As I feared, 'make' does not work locally because it uses a style file that
is not in my latex package. It fails on subcaption.sty.  I found  
subcaption.sty
on the web and put it in the directory with my other .sty files, but now it
wants a caption.sty file, because subcaption.sty depends on caption.sty.
I'm not going to redo my latex installation for this.  Is there
a reason why the requisite style files are not bundled with the distribution?

Also, I have a fork in my phartigan/ObservingStrategy directory on github.
But it is before any of the text was added to the yso section. It is 266
commits behind the LSST master. I navigated to the LSST ObservingStrategy
directory and clicked on fork, and it looks like it did something, and
puts me back in my phartigan directory. But the ObservingStrategy is
not updated. I don't see how to delete my existing ObservingStrategy or
to overwrite it.

I did glance through the merged YSO section that is on the LSST site.
The two yso contributions did not merge smoothly and I don't think
it reads very well. I would give a try at improving it,
but must solve these github issues first.

ph

[Hide Quoted Text]
Hmm.

As these are all latex documents, to see how things look presumably I'll run
latex on some master file that sources the various .tex and uses some style
file that it finds in the repository. I guess that is what `make' does?
And the output is a .dvi or .eps or .pdf? Does my local latex version  
matter, or
is that bundled in the repo as well, with the fonts and everything?

So the purpose of the fork is that I own it and can put edits there.  
Ok, but then
why not just edit on the fork directly, why clone? Or can't I run  
'make' directly
on the fork, just on a local clone? Or is it the convenience of using a real
editor and unix commands and not some git-gui with green buttons on the web?

ph
No, you've got it Steve. The only difference is that Pat, you should clone
using the .git address of your fork,

    git clone git@github.com:phartigan/ObservingStrategy.git

and then push your edits there, so you can submit a pull request on the
GitHub site. To keep up with the base repo, you'll want to connect to that
as well, via

    git remote add base git@github.com:LSSTScienceColl
aborations/ObservingStrategy.git

Then you can

    git pull base master

to bring your local repository up to date with the primary version, and

    git push origin master

to publish your edits on your fork.

On Wed, Apr 27, 2016 at 12:48 PM, StephenRidgway <notifications@github.com>
wrote:
Hi,

I’m probably the least adept with GIT of any of you. I don’t know if our
setups are similar, but when I get confused I find it clears things up to
delete my current files and start over, as follows:


Start in github directory.

git clone git@github.com:LSSTScienceCollaborations/ObservingStrategy.git

edit files in the “whitepaper” directory


To typeset locally, use "make" in the whitepaper directory.


git commit -am "comment"


git status


git push origin master


If I’m doing anything dangerous here, hopefully somebody will point it
out, but it seems to do what I need.

Steve
On Apr 27, 2016, at 12:35 PM, phartigan <notifications@github.com>
wrote:
ok. Am about done with the video. I'll probably give the figures a shot.
I guess I make a fork, and then I edit, commit, and should see
changes, and then
submit a pull request. My understanding is that clones are only needed if
I'm working locally off-line. Doesn't look like I need to create
branches.
Is that correct?

ph
Hi @phartigan - just to reply to your question from a few comments
ago: I do not expect to touch youngstars.tex again today, I will be
working on Chapter 4 issues.

If you are comfortable enough with github to e.g. change figure
boundaries or add the authors to references.bib , please go ahead
and do so.

---
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub:
https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-215190543
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub <
https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-215203790
—
You are receiving this because you commented.
Reply to this email directly or view it on GitHub
<https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-215207133>
---

**Comment by lmwalkowicz** [comment]
Hey Pat,
Going over the text and editing it is actually on my plate at the moment--
I took it on at the editorial meeting and will be working on it this week.
Cheers
L

**Comment by pmmcgehee** [comment]
That's great,  Lucianne...

Peregrine

On Mon, May 2, 2016 at 6:04 PM, Lucianne Walkowicz <notifications@github.com
> wrote:

> Hey Pat,
> Going over the text and editing it is actually on my plate at the moment--
> I took it on at the editorial meeting and will be working on it this week.
> Cheers
> L
>
>
> On Mon, May 2, 2016 at 11:00 AM phartigan <notifications@github.com>
> wrote:
>
> > Hi,
> >
> > A couple of questions...
> >
> > I seem to have the ssh-keygen working. and successfully downloaded my
> > github fork
> > to my local machine with the clone command. However...
> >
> > As I feared, 'make' does not work locally because it uses a style file
> that
> > is not in my latex package. It fails on subcaption.sty. I found
> > subcaption.sty
> > on the web and put it in the directory with my other .sty files, but now
> it
> > wants a caption.sty file, because subcaption.sty depends on caption.sty.
> > I'm not going to redo my latex installation for this. Is there
> > a reason why the requisite style files are not bundled with the
> > distribution?
> >
> > Also, I have a fork in my phartigan/ObservingStrategy directory on
> github.
> > But it is before any of the text was added to the yso section. It is 266
> > commits behind the LSST master. I navigated to the LSST ObservingStrategy
> > directory and clicked on fork, and it looks like it did something, and
> > puts me back in my phartigan directory. But the ObservingStrategy is
> > not updated. I don't see how to delete my existing ObservingStrategy or
> > to overwrite it.
> >
> > I did glance through the merged YSO section that is on the LSST site.
> > The two yso contributions did not merge smoothly and I don't think
> > it reads very well. I would give a try at improving it,
> > but must solve these github issues first.
> >
> > ph
> >
> > [Hide Quoted Text]
> > Hmm.
> >
> > As these are all latex documents, to see how things look presumably I'll
> > run
> > latex on some master file that sources the various .tex and uses some
> style
> > file that it finds in the repository. I guess that is what `make' does?
> > And the output is a .dvi or .eps or .pdf? Does my local latex version
> > matter, or
> > is that bundled in the repo as well, with the fonts and everything?
> >
> > So the purpose of the fork is that I own it and can put edits there.
> > Ok, but then
> > why not just edit on the fork directly, why clone? Or can't I run
> > 'make' directly
> > on the fork, just on a local clone? Or is it the convenience of using a
> > real
> > editor and unix commands and not some git-gui with green buttons on the
> > web?
> >
> > ph
> > No, you've got it Steve. The only difference is that Pat, you should
> clone
> > using the .git address of your fork,
> >
> > git clone git@github.com:phartigan/ObservingStrategy.git
> >
> > and then push your edits there, so you can submit a pull request on the
> > GitHub site. To keep up with the base repo, you'll want to connect to
> that
> > as well, via
> >
> > git remote add base git@github.com:LSSTScienceColl
> > aborations/ObservingStrategy.git
> >
> > Then you can
> >
> > git pull base master
> >
> > to bring your local repository up to date with the primary version, and
> >
> > git push origin master
> >
> > to publish your edits on your fork.
> >
> > On Wed, Apr 27, 2016 at 12:48 PM, StephenRidgway <
> notifications@github.com
> > >
> > wrote:
> > Hi,
> >
> > I’m probably the least adept with GIT of any of you. I don’t know if our
> > setups are similar, but when I get confused I find it clears things up to
> > delete my current files and start over, as follows:
> >
> >
> > Start in github directory.
> >
> > git clone git@github.com:LSSTScienceCollaborations/ObservingStrategy.git
> >
> > edit files in the “whitepaper” directory
> >
> >
> > To typeset locally, use "make" in the whitepaper directory.
> >
> >
> > git commit -am "comment"
> >
> >
> > git status
> >
> >
> > git push origin master
> >
> >
> > If I’m doing anything dangerous here, hopefully somebody will point it
> > out, but it seems to do what I need.
> >
> > Steve
> > On Apr 27, 2016, at 12:35 PM, phartigan <notifications@github.com>
> > wrote:
> > ok. Am about done with the video. I'll probably give the figures a shot.
> > I guess I make a fork, and then I edit, commit, and should see
> > changes, and then
> > submit a pull request. My understanding is that clones are only needed if
> > I'm working locally off-line. Doesn't look like I need to create
> > branches.
> > Is that correct?
> >
> > ph
> > Hi @phartigan - just to reply to your question from a few comments
> > ago: I do not expect to touch youngstars.tex again today, I will be
> > working on Chapter 4 issues.
> >
> > If you are comfortable enough with github to e.g. change figure
> > boundaries or add the authors to references.bib , please go ahead
> > and do so.
> >
> > ---
> > You are receiving this because you were mentioned.
> > Reply to this email directly or view it on GitHub:
> >
> >
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-215190543
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly or view it on GitHub <
> >
> >
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-215203790
> > —
> > You are receiving this because you commented.
> > Reply to this email directly or view it on GitHub
> > <
> >
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-215207133
> > >
> > ---
> > You are receiving this because you were mentioned.
> > Reply to this email directly or view it on GitHub:
> >
> >
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-215225
> >
> > —
> > You are receiving this because you were assigned.
> >
> >
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-216276799
> >
>
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-216277757>
>



-- 
Dr. Peregrine M. McGehee
Santa Clarita, CA    (626) 993-4199

**Comment by phartigan** [comment]
sounds good. What's there looked like blocks of text and did not flow, but
if the merge is not done then I won't worry about it. Looks like I have
more to learn/setup anyway before I can incorporate changes.

My sense is that FU Oris and the like will be found as a matter of  
course with any
standard cadence. I think the main contribution will be that LSST will be
able to routinely follow >10^4 TTS at least in the sky. Given that
FU Oris are supposed to go off every 10^4 years or so, we ought to be able
to put statistics on the frequencies of outbursts, watch them decay, etc.

ph

> Hey Pat,
> Going over the text and editing it is actually on my plate at the moment--
> I took it on at the editorial meeting and will be working on it this week.
> Cheers
> L
>

**Comment by pmmcgehee** [comment]
There will be similar statistics for the EXors, which have outburst
timescales more on the order of years.

P

On Mon, May 2, 2016 at 7:13 PM, phartigan <notifications@github.com> wrote:

>
> sounds good. What's there looked like blocks of text and did not flow, but
> if the merge is not done then I won't worry about it. Looks like I have
> more to learn/setup anyway before I can incorporate changes.
>
> My sense is that FU Oris and the like will be found as a matter of
> course with any
> standard cadence. I think the main contribution will be that LSST will be
> able to routinely follow >10^4 TTS at least in the sky. Given that
> FU Oris are supposed to go off every 10^4 years or so, we ought to be able
> to put statistics on the frequencies of outbursts, watch them decay, etc.
>
> ph
>
>
> > Hey Pat,
> > Going over the text and editing it is actually on my plate at the
> moment--
> > I took it on at the editorial meeting and will be working on it this
> week.
> > Cheers
> > L
> >
> > On Mon, May 2, 2016 at 11:00 AM phartigan <notifications@github.com>
> wrote:
> >
> >> Hi,
> >>
> >> A couple of questions...
> >>
> >> I seem to have the ssh-keygen working. and successfully downloaded my
> >> github fork
> >> to my local machine with the clone command. However...
> >>
> >> As I feared, 'make' does not work locally because it uses a style file
> that
> >> is not in my latex package. It fails on subcaption.sty. I found
> >> subcaption.sty
> >> on the web and put it in the directory with my other .sty files, but
> now it
> >> wants a caption.sty file, because subcaption.sty depends on caption.sty.
> >> I'm not going to redo my latex installation for this. Is there
> >> a reason why the requisite style files are not bundled with the
> >> distribution?
> >>
> >> Also, I have a fork in my phartigan/ObservingStrategy directory on
> github.
> >> But it is before any of the text was added to the yso section. It is 266
> >> commits behind the LSST master. I navigated to the LSST
> ObservingStrategy
> >> directory and clicked on fork, and it looks like it did something, and
> >> puts me back in my phartigan directory. But the ObservingStrategy is
> >> not updated. I don't see how to delete my existing ObservingStrategy or
> >> to overwrite it.
> >>
> >> I did glance through the merged YSO section that is on the LSST site.
> >> The two yso contributions did not merge smoothly and I don't think
> >> it reads very well. I would give a try at improving it,
> >> but must solve these github issues first.
> >>
> >> ph
> >>
> >> [Hide Quoted Text]
> >> Hmm.
> >>
> >> As these are all latex documents, to see how things look presumably I'll
> >> run
> >> latex on some master file that sources the various .tex and uses some
> style
> >> file that it finds in the repository. I guess that is what `make' does?
> >> And the output is a .dvi or .eps or .pdf? Does my local latex version
> >> matter, or
> >> is that bundled in the repo as well, with the fonts and everything?
> >>
> >> So the purpose of the fork is that I own it and can put edits there.
> >> Ok, but then
> >> why not just edit on the fork directly, why clone? Or can't I run
> >> 'make' directly
> >> on the fork, just on a local clone? Or is it the convenience of using a
> >> real
> >> editor and unix commands and not some git-gui with green buttons on the
> >> web?
> >>
> >> ph
> >> No, you've got it Steve. The only difference is that Pat, you should
> clone
> >> using the .git address of your fork,
> >>
> >> git clone git@github.com:phartigan/ObservingStrategy.git
> >>
> >> and then push your edits there, so you can submit a pull request on the
> >> GitHub site. To keep up with the base repo, you'll want to connect to
> that
> >> as well, via
> >>
> >> git remote add base git@github.com:LSSTScienceColl
> >> aborations/ObservingStrategy.git
> >>
> >> Then you can
> >>
> >> git pull base master
> >>
> >> to bring your local repository up to date with the primary version, and
> >>
> >> git push origin master
> >>
> >> to publish your edits on your fork.
> >>
> >> On Wed, Apr 27, 2016 at 12:48 PM, StephenRidgway <
> notifications@github.com
> >> >
> >> wrote:
> >> Hi,
> >>
> >> I’m probably the least adept with GIT of any of you. I don’t know if our
> >> setups are similar, but when I get confused I find it clears things up
> to
> >> delete my current files and start over, as follows:
> >>
> >>
> >> Start in github directory.
> >>
> >> git clone git@github.com:
> LSSTScienceCollaborations/ObservingStrategy.git
> >>
> >> edit files in the “whitepaper” directory
> >>
> >>
> >> To typeset locally, use "make" in the whitepaper directory.
> >>
> >>
> >> git commit -am "comment"
> >>
> >>
> >> git status
> >>
> >>
> >> git push origin master
> >>
> >>
> >> If I’m doing anything dangerous here, hopefully somebody will point it
> >> out, but it seems to do what I need.
> >>
> >> Steve
> >> On Apr 27, 2016, at 12:35 PM, phartigan <notifications@github.com>
> >> wrote:
> >> ok. Am about done with the video. I'll probably give the figures a shot.
> >> I guess I make a fork, and then I edit, commit, and should see
> >> changes, and then
> >> submit a pull request. My understanding is that clones are only needed
> if
> >> I'm working locally off-line. Doesn't look like I need to create
> >> branches.
> >> Is that correct?
> >>
> >> ph
> >> Hi @phartigan - just to reply to your question from a few comments
> >> ago: I do not expect to touch youngstars.tex again today, I will be
> >> working on Chapter 4 issues.
> >>
> >> If you are comfortable enough with github to e.g. change figure
> >> boundaries or add the authors to references.bib , please go ahead
> >> and do so.
> >>
> >> ---
> >> You are receiving this because you were mentioned.
> >> Reply to this email directly or view it on GitHub:
> >>
> >>
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-215190543
> >> —
> >> You are receiving this because you are subscribed to this thread.
> >> Reply to this email directly or view it on GitHub <
> >>
> >>
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-215203790
> >> —
> >> You are receiving this because you commented.
> >> Reply to this email directly or view it on GitHub
> >> <
> >>
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-215207133
> >> >
> >> ---
> >> You are receiving this because you were mentioned.
> >> Reply to this email directly or view it on GitHub:
> >>
> >>
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-215225
> >>
> >> —
> >> You are receiving this because you were assigned.
> >>
> >>
> >> Reply to this email directly or view it on GitHub
> >> <
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-216276799
> >
> >>
> >
> >
> > ---
> > You are receiving this because you were mentioned.
> > Reply to this email directly or view it on GitHub:
> >
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-216277757
>
>
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-216297091>
>



-- 
Dr. Peregrine M. McGehee
Santa Clarita, CA    (626) 993-4199

**Comment by rhiannonlynne** [comment]
Re: Updating your fork of the LSSTScienceCollaborations/ObservingStrategy
whitepaper

You can do this in two ways.
1 - If you do not have content in your fork which is different from the
main repository, and you don't care about your fork, you could delete your
fork (go to your fork's repository on github, go to 'Settings', then look
for "Delete Repository". You will get a big warning, but you can delete
your repository, which deletes your fork. Make sure you also delete any
cloned copies of the repository as well.)  Then hit the "Fork" button on
the original repository ( LSSTScienceCollaborations/ObservingStrategy), to
redo the whole fork. You will need to do a brand-new 'git clone' on your
local machine.
(simply hitting "fork" on the main repository doesn't work when you've
already forked it once .. think of it like an "initialize the fork" kind of
button).

2 - You can also do a PR (Pull Request) from the main repository into your
fork of the repository (basically, the reverse of what you did when you
submitted your PR to the main repo). The only tricky thing here is making
sure that you are requesting the PR in the "right direction". You should be
able to tell from the list of commits that the PR is asking to merge.
Basically:
  * go to https://github.com/LSSTScienceCollaborations/ObservingStrategy
  * click "New PR", then click "Compare across forks" on the next page
  * choose "base fork" = your fork, and "head fork" =
LSSTScienceCollaborations/ObservingStrategy   (choose "master" branch on
both)
  * look at the commits that will be merged into your fork (you should see
commits that reflect the work that's gone on since the last time you merged)
  * If it looks right, enter a message for the commit ('pull in updates
from upstream'), and click "Create Pull Request"
  * click "merge pull request" and then "confirm merge"
(note that you should see a message saying that there are no merge
conflicts .. if there are conflicts, you need to do some more work and you
might need some further advice, because what this means is your copy has
changes that never got into the LSSTScienceCollaborations/ObservingStrategy
fork).
Then go to your local-on-disk clones, and do 'git pull'.

Lynne

**Comment by phartigan** [comment]
yes, and both should be found easily

> There will be similar statistics for the EXors, which have outburst
> timescales more on the order of years.
>
> P

**Comment by phartigan** [comment]
Thanks, I'll try this. I was reforking, and as you note, it was unhappy.

> Re: Updating your fork of the LSSTScienceCollaborations/ObservingStrategy
> whitepaper
>
> You can do this in two ways.
> 1 - If you do not have content in your fork which is different from the
> main repository, and you don't care about your fork, you could delete your
> fork (go to your fork's repository on github, go to 'Settings', then look
> for "Delete Repository". You will get a big warning, but you can delete
> your repository, which deletes your fork. Make sure you also delete any
> cloned copies of the repository as well.)  Then hit the "Fork" button on
> the original repository ( LSSTScienceCollaborations/ObservingStrategy), to
> redo the whole fork. You will need to do a brand-new 'git clone' on your
> local machine.
> (simply hitting "fork" on the main repository doesn't work when you've
> already forked it once .. think of it like an "initialize the fork" kind of
> button).
>
> 2 - You can also do a PR (Pull Request) from the main repository into your
> fork of the repository (basically, the reverse of what you did when you
> submitted your PR to the main repo). The only tricky thing here is making
> sure that you are requesting the PR in the "right direction". You should be
> able to tell from the list of commits that the PR is asking to merge.
> Basically:
>   * go to https://github.com/LSSTScienceCollaborations/ObservingStrategy
>   * click "New PR", then click "Compare across forks" on the next page
>   * choose "base fork" = your fork, and "head fork" =
> LSSTScienceCollaborations/ObservingStrategy   (choose "master" branch on
> both)
>   * look at the commits that will be merged into your fork (you should see
> commits that reflect the work that's gone on since the last time you merged)
>   * If it looks right, enter a message for the commit ('pull in updates
> from upstream'), and click "Create Pull Request"
>   * click "merge pull request" and then "confirm merge"
> (note that you should see a message saying that there are no merge
> conflicts .. if there are conflicts, you need to do some more work and you
> might need some further advice, because what this means is your copy has
> changes that never got into the LSSTScienceCollaborations/ObservingStrategy
> fork).
> Then go to your local-on-disk clones, and do 'git pull'.
>
> Lynne
>

**Comment by pmmcgehee** [comment]
true enough... although characterization of TTS color variations might be
problematic due to complex and irregular variability.

On Mon, May 2, 2016 at 10:04 PM, phartigan <notifications@github.com> wrote:

>
> yes, and both should be found easily
>
> > There will be similar statistics for the EXors, which have outburst
> > timescales more on the order of years.
> >
> > P
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-216346758>
>



-- 
Dr. Peregrine M. McGehee
Santa Clarita, CA    (626) 993-4199

**Comment by phartigan** [comment]
I don't expect the colors to behave necessarily so they can be picked out
in some clever color-color-amplitude-time diagram. But if they are in  
a star formation
region and they suddenly brighten by 2 magnitudes we will see them easily
With LSST. The LSST depth is such that enough such objects are being monitored
that we ought to finally be able to get some statistics on the longevity
and frequency of this bursting activity.


> true enough... although characterization of TTS color variations might be
> problematic due to complex and irregular variability.
>

**Comment by phartigan** [comment]
Hi,

I created a summary web page for how to use GitHub with LSST
for a first-time user.

http://sparky.rice.edu/~hartigan/github.html

This is mainly for me, but perhaps it could be useful to
others. Let me know if you see any glaring errors.

Pat

**Comment by drphilmarshall** [comment]
Nice, Pat - thank you! I like your diagram a lot, I often act that out by
hand waving when trying to explain git and GitHub to people. I should copy
it and put it in the Getting Started FAQ. I guess I would make a
distinction between "clone" and "pull" though, the latter is really the
opposite of "push", while "clone" is what you do just once, to get a local
copy of the repo in the first place.

**Comment by drphilmarshall** [comment]
Looks like @lmwalkowicz did not have time to merge the text together any better that I did already, so let's close this and see what the section's reviewer makes of it. Perhaps it will pass muster! Thanks for all your contributions - if you want to take on the editing job of removing all remaining redundancy, please let me know.

**Comment by phartigan** [comment]
Would you like me to do it?  Now is not a bad time for me.

ph

**Comment by pmmcgehee** [comment]
Thanks for volunteering, Pat. I can also it over if you wish.

Peregrine

**Comment by AshishMahabal** [comment]
Pat/Peregrine, please go for it!

**Comment by phartigan** [comment]
ok. give me a day and I'll see what I can do with it and then we
can iterate if need be. Which sections, specifically, need to be
downloaded and merged?

ph

> Thanks for volunteering, Pat. I can also it over if you wish.
>
> Peregrine

**Comment by pmmcgehee** [comment]
Hi Pat,

I'll have a look over your first pass.

Thanks,
Peregrine

On Mon, May 23, 2016 at 10:55 PM, phartigan <notifications@github.com>
wrote:

>
> ok. give me a day and I'll see what I can do with it and then we
> can iterate if need be. Which sections, specifically, need to be
> downloaded and merged?
>
> ph
>
> > Thanks for volunteering, Pat. I can also it over if you wish.
> >
> > Peregrine
> > On May 23, 2016 22:04, "phartigan" <notifications@github.com> wrote:
> >
> >>
> >> Would you like me to do it? Now is not a bad time for me.
> >>
> >> ph
> >>
> >> > Looks like @lmwalkowicz did not have time to merge the text together
> >> > any better that I did already, so let's close this and see what the
> >> > section's reviewer makes of it. Perhaps it will pass muster! Thanks
> >> > for all your contributions - if you want to take on the editing job
> >> > of removing all remaining redundancy, please let me know.
> >> >
> >> > ---
> >> > You are receiving this because you were mentioned.
> >> > Reply to this email directly or view it on GitHub:
> >> >
> >>
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-221166078
> >>
> >>
> >>
> >> —
> >> You are receiving this because you were mentioned.
> >> Reply to this email directly or view it on GitHub
> >> <
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-221167389
> >
> >>
> >
> >
> > ---
> > You are receiving this because you were mentioned.
> > Reply to this email directly or view it on GitHub:
> >
> https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-221167803
>
>
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/321#issuecomment-221173520>
>



-- 
Dr. Peregrine M. McGehee
Santa Clarita, CA    (626) 993-4199
